### ISSUE TITLE
Getting the  tokens

### ISSUE THREAD

**Issue by LCB0B** [Berkanktk/BeReal-Fetcher#3]
hey, for  _'8. Now, go into the recorded session and locate https://securetoken.googleapis.com/ '_

shouldn't it be 'https://firebaselogging-pa.googleapis.com' ? At least I don't find 'https://securetoken.googleapis.com/ ' :)

**Comment by LCB0B** [comment]
or rather https://fcmtoken.googleapis.com ?

**Comment by Berkanktk** [comment]
> hey, for _'8. Now, go into the recorded session and locate https://securetoken.googleapis.com/ '_
> 
> shouldn't it be 'https://firebaselogging-pa.googleapis.com' ? At least I don't find 'https://securetoken.googleapis.com/ ' :)

I don't think you can find the `SECURE_TOKEN` and `REFRESH_TOKEN` in that one. I also remember that it wasn't in every session i got the https://securetoken.googleapis.com/. So maybe try to see if you can run it again, and be lucky to see it? :)

**Comment by LCB0B** [comment]
I tried a lot of times, and I never get the https://securetoken.googleapis.com/. 
I think they replaced it with https://fcmtoken.googleapis.com/ ?

**Comment by noahrepublic** [comment]
> I tried a lot of times, and I never get the https://securetoken.googleapis.com/. I think they replaced it with https://fcmtoken.googleapis.com/ ?

Any update on this?
